src/System.Linq/src/System/Linq/Enumerable.cs

```diff
@@ -703,11 +709,13 @@ public static IEnumerable<TSource> SkipWhile<TSource>(this IEnumerable<TSource>
 
         private static IEnumerable<TSource> SkipWhileIterator<TSource>(IEnumerable<TSource> source, Func<TSource, bool> predicate)
         {
-            bool yielding = false;
-            foreach (TSource element in source)
+            using (IEnumerator<TSource> e = source.GetEnumerator())
```

Comment:
Replace `e` with `element` (or vice versa below). This is causing compilation error [on CI](http://dotnet-ci.cloudapp.net/job/dotnet_corefx_mac_release_prtest/1561/consoleFull#-575944600a82fefab-f698-416f-8fca-58544c94cd4e).